csrc/mmdeploy/codebase/mmpose/topdown_affine.cpp

```diff
@@ -32,6 +32,11 @@ class TopDownAffine : public transform::Transform {
     stream_ = args["context"]["stream"].get<Stream>();
     assert(args.contains("image_size"));
     from_value(args["image_size"], image_size_);
+    use_warp_affine_ = args.value("use_warp_affine_", use_warp_affine_);
```

Comment:
In what case there is `use_warp_affine_` in `TopDownAffine` config?

Reply:
the purpose of add `use_warp_affine_` is to replace `warpAffine` with `crop` and `resize` which will be a little bit faster.

Reply:
what's `use_udp_`? When will it be set to true and when will not?

Reply:
It's a [option](https://mmpose.readthedocs.io/en/latest/api.html#mmpose.datasets.pipelines.top_down_transform.TopDownAffine) whether use `Unbiased data processing` for the input data.

Some model config set this to True, like [this](https://github.com/open-mmlab/mmpose/blob/8174b019e1553ffe91a959883cd46de44c0340d8/configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/hrnetv2_w64_coco_384x288_udp.py#L79)